src/test/scala/org/kibanaLoadTest/scenario/Dashboard.scala

```diff
@@ -3,132 +3,173 @@ package org.kibanaLoadTest.scenario
-        )
-        .exec(session =>
-          //convert Vector -> String
+        ).exec(session =>
+          //convert Vector -> String for visualizations request
           session.set(
             "vizListString",
             session("vizVector").as[Seq[String]].mkString(",")
           )
-        )
-        .exec(session => {
-          //convert Vector -> String
+        ).exec(session =>
+          //convert Vector -> String for search&map request
           session.set(
             "searchAndMapString",
             session("searchAndMapVector").as[Seq[String]].mkString(",")
           )
-        })
-        .exec(
+        ).exec(
           http("query visualizations")
             .post("/api/saved_objects/_bulk_get")
-            .body(
-              StringBody(
-                "[" +
-                  "${vizListString}"
-                    .concat(
-                      ", { \"id\":\"${indexPatternId}\", \"type\":\"index-pattern\"  }]"
-                    )
-              )
-            )
+            .body(StringBody("[${vizListString}, {\"id\":\"${indexPatternId}\", \"type\":\"index-pattern\"}]"))
             .asJson
             .headers(headers)
             .header("Referer", baseUrl + "/app/dashboards")
             .check(status.is(200))
-        )
-        .exec(
+        ).exec(
           http("query search & map")
             .post("/api/saved_objects/_bulk_get")
             .body(StringBody("""[ ${searchAndMapString} ]""".stripMargin))
             .asJson
             .headers(headers)
             .header("Referer", baseUrl + "/app/dashboards")
             .check(status.is(200))
-        )
-        .exec(
+        ).exec(
+          http("query index pattern meta fields")
+          .get("/api/index_patterns/_fields_for_wildcard")
+            .queryParam("pattern", "kibana_sample_data_ecommerce")
+            .queryParam("meta_fields", "_source")
+            .queryParam("meta_fields", "_id")
+            .queryParam("meta_fields", "_type")
+            .queryParam("meta_fields", "_index")
+            .queryParam("meta_fields", "_score")
+            .headers(headers)
+            .header("Referer", baseUrl + "/app/dashboards")
+            .check(status.is(200))
+        ).exec(
+          http("query index pattern search fields")
+            .get("/api/saved_objects/_find")
+            .queryParam("fields", "title")
+            .queryParam("per_page", "10")
+            .queryParam("search", "kibana_sample_data_ecommerce")
+            .queryParam("search_fields", "title")
+            .queryParam("type", "index-pattern")
+            .headers(headers)
+            .header("Referer", baseUrl + "/app/dashboards")
+            .check(status.is(200))
+        ).exec(
+          http("query input control settings")
+            .get("/api/input_control_vis/settings")
+            .headers(headers)
+            .header("Referer", baseUrl + "/app/dashboards")
+            .check(status.is(200))
```

Comment:
This request is called 4 times (completely identical calls):

![image](https://user-images.githubusercontent.com/10977896/113866619-d2667b00-97ad-11eb-9132-2e0d07ce2b86.png)
I wonder if it is something we can optimise and should we repeat it load testing as well?